smac/tae/execute_ta_run.py

```diff
@@ -149,6 +149,12 @@ def start(self, config:Configuration,
                                                           seed=seed,
                                                           instance_specific=instance_specific)
 
+        # Catch NaN or inf.
+        if (self.run_obj == 'runtime' and not np.isfinite(runtime) or
+            self.run_obj == 'quality' and not np.isfinite(cost)):
+            status = StatusType.CRASHED
+            runtime, cost = 0, 0
```

Comment:
Adding a cost of `0` has led to some issues in Java-SMAC. Not sure if we should go this way. Also, we have observed a runtime which could be used here.